pizzeria-backend/Controllers/ProductsController.cs

```diff
@@ -68,6 +41,18 @@ public async Task<IActionResult> GetProduct(int id)
             return Ok(pr);
         }
 
+        [HttpGet("menu")]
+        [Produces("application/json")]
+        public async Task<IActionResult> GetMenu()
+        {
+            var menu = await _productService.GetMenu();
+            if (menu == null)
+            {
+                return NotFound("message: " + "There are no items in the menu");
```

Comment:
Why is this "message: " needed?

Reply:
Not needed, only had a thought it would look good

Reply:
Done